### PR TITLE
DOCS-1965: Update intermode tutorial for R2D2

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tutorials/custom/controlling-an-intermode-rover-canbus.md [hunk 1]
```diff
@@ -12,18 +12,18 @@ tags:
     "CAN bus",
     "Intermode",
   ]
-description: "Integrate an Intermode rover as a modular-resource-based component with CAN bus."
+description: "Integrate an Intermode rover as a modular resourcebase component with CAN bus."
```
Comment (sguequierre) [inline docs/tutorials/custom/controlling-an-intermode-rover-canbus.md:15]:
```suggestion
description: "Integrate an Intermode rover as a modular resource base component with CAN bus."
```
I think?

File: docs/tutorials/custom/controlling-an-intermode-rover-canbus.md
Comment (sguequierre) [inline docs/tutorials/custom/controlling-an-intermode-rover-canbus.md:1]: Out of diff but needs setup instructions updated @ line 277

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/tutorials/custom/controlling-an-intermode-rover-canbus.md
Comment (npentrel) [reply docs/tutorials/custom/controlling-an-intermode-rover-canbus.md:1]: fixed - I think this tutorial needs more attention but I'll keep that for a separate PR (there's another ticket for it)

File: docs/tutorials/custom/controlling-an-intermode-rover-canbus.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ sudo apt install git
 
 ### Configure the Intermode base resource
 
-If you have not already, first create a new machine in the [Viam app](https://app.viam.com/) and follow the instructions in the **Setup** tab to connect the machine to the cloud.
+If you have not already, first add a new machine in the [Viam app](https://app.viam.com/) and follow the setup instructions to connect the machine to the cloud.
```
Comment (sguequierre) [inline docs/tutorials/custom/controlling-an-intermode-rover-canbus.md:277]:
```suggestion
If you have not already, first add a new machine in the [Viam app](https://app.viam.com/) and follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} to connect the machine to the cloud.
```
(opt)

**Review by sguequierre — APPROVED** [review]
lgtm!

**Pull request merged**